extensions/amp-mraid/0.1/amp-mraid.js

```diff
@@ -85,6 +80,12 @@ export class MraidInitializer {
         (element.getAttribute(FALLBACK_ON) || '').split(' ');
     this.fallback_ = fallbackOn.includes(FallbackErrorNames.MISMATCH);
 
+    if (getMode().runtime !== 'inabox') {
+      dev().fine(TAG, 'Only supported with Inabox');
+      handleMismatch();
```

Comment:
`this.handleMismatch_()`? 
Also is fallback_on a required attribute? What happens if this.fallback_ is false?

Reply:
The attribute is optional. You would leave it off (and so have `fallback_` being false) if you're confident that you're serving to a mobile app environment and so never want to fall back to web apis.

For example, if I know from the user agent or query string on the ad request that I'm serving to a mobile app, then even if mraid fails to load I don't want to fall back to intersection observer because that would give incorrect visibility values.

Reply:
But I'm worrying that if the ad network doesn't do anything to the `<amp-mraid>` on the page, that eventually leads to the ad creative not working within an AMP page. 

Which kind of leads us back to the same issue why we want to make `<amp-mraid>` valid at first place.

Reply:
> `this.handleMismatch_()?`

whoops, fixed!

> I'm worrying that if the ad network doesn't do anything to the `<amp-mraid>` on the page, that eventually leads to the ad creative not working within an AMP page.

If you're a creative author putting `<amp-mraid>` in your creatives without knowing where they'll be served, you want `fallback=mismatch`.  I'd be fine making that the default if you'd prefer that?

With the change in this PR, AMP creatives with amp-mraid work correctly on AMP pages (by not using mraid).

Reply:
Yes, making fallback=mismatch should work. So we will be supporting something like : 
`<amp-mraid>` equals to `<amp-mraid falllback=mismatch>`

Reply:
changed this to be the default in #21738